sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql

```diff
@@ -25,6 +28,7 @@ WITH stage_1 AS (
   GROUP BY
     subscription_start_date,
     country_name,
+    country,
```

Comment:
Group by column order doesn't really matter, but for consistency with the above change...
```suggestion
    country,
    country_name,
```